### DISCUSSION TITLE
passt usermode networking

### DISCUSSION THREAD

**Discussion by stuaxo** [dosemu2/dosemu2#1813] [Ideas]
Hi,
   passt looks like an interesting development for networking, aiming itself as a replacement to slirp, being taken up by qemu and podman -

http://blog.vmsplice.net/2021/10/a-new-approach-to-usermode-networking.html

https://github.com/containers/podman/pull/16141

Cheers
Stuart

**Comment by stsp** [comment]
Yes, this is indeed very interesting!
Its not obvious for me whether they
introduce something new, or its just
a more carefully written same slirp
concept. I guess they are proud by
the fact that they re-used the host's
TCP queues and represend guest
acknowledges more properly, but I
think conceptually its the same thing.

Anyway.
We need to wait for ubuntu packages
of that. Also I don't see a proper user
docs with API description.
